### PR TITLE
Refactor & Add Comments to create.js.erb; Style Fixes for Question Comments & Toolbar Buttons

### PR DESCRIPTION
**Pull request by noi5e**
1. Attach image upload functionality ONLY to edit comment form in comment responses.
2. Style fixes for toolbar buttons (make colors all the same, `cursor: pointer;`
3. Correct the width of comments on `question/show` partial

---
(This issue is part of the larger Comment Editor Overhaul Project with Outreachy. Refer to Planning Issue #9069 for more context)

### CODE REVIEW

File: app/views/comments/create.js.erb [hunk 1]
```diff
@@ -1,23 +1,43 @@
+// this code runs after a new comment is posted.
+//   1. it inserts the new comment into the DOM
+//   2. it also creates image upload functionality for the new comment form (see /app/assets/javascripts/editorToolbar.js)
```
Comment (noi5e) [inline app/views/comments/create.js.erb:3]: Explanatory comment.

File: app/views/comments/create.js.erb [hunk 2]
```diff
@@ -1,23 +1,43 @@
...
+  fileUploadElements = fileUploadElements.concat([
+    {
+      selector: '#comment-form-body-reply-<%= @comment.cid %>',
+      isButton: false
+    },
+    {
+      selector: '#dropzone-small-reply-<%= @comment.cid %>',
+      isButton: true
+    }
+  ]);
```
Comment (noi5e) [inline app/views/comments/create.js.erb:30]: Reminder that this code is for attaching image upload functionality to fresh comments.

I realized that a fresh comment can either:
1. Be a comment NOT responding to another comment (will have both an edit form and a reply form).
2. Be in response to another comment (will have ONLY an edit form).

Therefore, if a comment is a response, image upload functionality should only be attached to the edit form (the reply form doesn't exist).

That's the main change I'm making in this PR.

File: app/views/questions/show.html.erb [hunk 3]
```diff
@@ -58,34 +58,36 @@
...
 
   <% comments = @node.comments_viewable_by(current_user) %>
-  <h3><span id="comment-count"><%= comments.size %></span> Comments</h3>
-  <div id="comments-list">
-  <%= javascript_include_tag "editorToolbar" %>
-  <%= javascript_include_tag "comment" %>
-    <% comments.includes([:node, :replied_comments]).order("timestamp ASC").each do |comment| %>
-      <% if comment.reply_to.nil? %>
-        <%= render :partial => "notes/comment", :locals => {:comment => comment} %>
+  <div id="legacy-editor-container" class="col-lg-10">
```
Comment (noi5e) [inline app/views/questions/show.html.erb:61]: Style fix here to shrink the width of question comment container to 75%... That's what it used to be before recent changes I can't track.

The partials for `question/show`, `wiki/show`, and `notes/show` could use some serious clean-up!

**Review by jywarren — APPROVED** [review]

**Comment by jywarren** [comment]
Nicely done. And, agree about the cleanup in the views; that code has seen MANY iterations!

**Pull request merged**